tests/test_estimate_descriptor.py

```diff
@@ -439,11 +439,9 @@ def test_descriptor_batch_bad_threshold_aggregation(self):
             with self.subTest(planVersion=descriptorVersion):
                 extractor = self.faceEngine.createFaceDescriptorEstimator(descriptorVersion)
                 descriptorBatch = self.getBatch(descriptorVersion, 2, DescriptorType.face)
-                with pytest.raises(LunaSDKException) as exceptionInfo:
-                    extractor.estimateDescriptorsBatch([faceWarp] * 2, aggregate=1, descriptorBatch=descriptorBatch)
-                self.assertLunaVlError(
-                    exceptionInfo,
-                    LunaVLError.BatchedInternalError.format(
-                        "Cant aggregate descriptors - all images'a GSs are less the threashold"
-                    ),
+                descriptorsRaw, descriptorAggregated = extractor.estimateDescriptorsBatch(
+                    [faceWarp] * 2, aggregate=1, descriptorBatch=descriptorBatch
                 )
+                for descriptorRaw in descriptorsRaw:
+                    self.assertDescriptor(descriptorVersion, descriptorRaw, DescriptorType.face)
```

Comment:
why is only one version of the descriptor used?

Reply:
added all descriptor versions